cmd/sidecar/local-server.go

```diff
@@ -169,7 +171,23 @@ func (srv *LocalServer) logHandler(w http.ResponseWriter, r *http.Request) {
 		log.Debugf("Log handler for '%s' received zero bytes.", actionId)
 		return
 	}
+	if ok {
+		root, _ := req.fnCtx.Callers.RootInstanceID()
```

Comment:
I think we have the power to ensure that root is always defined on the fnCtx, so that we won't have to perform this safety check here.